### PR TITLE
kiya 1.8.1 (new formula)

### PR DESCRIPTION
**Pull request by maikdiepenbroek**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by nandahkrishna — COMMENTED** [review]
Thanks for your PR, @maikdiepenbroek.

This looks like a nice application, but unfortunately it doesn't meet the criteria for inclusion into `homebrew/core` yet, as it fails the notability audit: 

```
kiya:
  * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
Error: 1 problem in 1 formula detected
```

(Thank you for not checking the last box in the PR template.)

This isn't a decision on the merits of the software, but we cannot package everything, so we must unfortunately draw a line on notability somewhere.

You may wish to maintain the Formula in a personal Tap for now. If you wish to, you may resubmit this in the future once it does meet the requirements. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

File: Formula/kiya.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  def install
+    ENV["GOPATH"] = buildpath
+    bin_path = buildpath/"src/github.com/kramphub/kiya"
+    bin_path.install buildpath.children
+
+    cd bin_path do
+      system "go", "build", "-o", bin/"kiya", "./cmd/kiya"
+      prefix.install_metafiles
+    end
```
Comment (nandahkrishna) [inline Formula/kiya.rb:18]: We're moving away from using `GOPATH`, you might want to check out some other `go`-based formulae like `tz` or `yq` to change this.

File: Formula/kiya.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  end
+
+  test do
+    system "#{bin}/kiya", "--version"
```
Comment (nandahkrishna) [inline Formula/kiya.rb:22]: We need a test other than checking for version or help output, as these tests rarely fail for broken installs. Something that tests the functionality of the application should be done in the `test do` block.

**Review by nandahkrishna — COMMENTED** [review]
Here are some suggestions to improve this formula.

**Comment by maikdiepenbroek** [comment]
I respect the decision, i wanted to submit it anyway knowing that the application did not meet the last requirement since i did not want to make it a custom tap. 

Thanks so much for the extensive feedback, i will perform the necessary changes and hope that the repository will gain enough stars/forks to make it into the core. It was a fun exercise nonetheless!

File: Formula/kiya.rb (hunk 1 above)
Comment (maikdiepenbroek) [reply Formula/kiya.rb:18]: Thanks for the suggestion, i've looked at a few but could not find other examples.

File: Formula/kiya.rb (hunk 2 above)
Comment (maikdiepenbroek) [reply Formula/kiya.rb:22]: Sure, fair enough.

**Pull request closed without merging**